test/unit/mongo/MongoTripodConfigTest.php

```diff
@@ -1380,6 +1380,78 @@ public function testDataLoadedInConfiguredDataSource()
         }
     }
 
+    public function testTransactionLogAndQueuesAreWrittenToCorrectDBAndCollection()
```

Comment:
that test looks reasonable.
- make sure theres nothing in the db
- do a write
- make sure its written into the correct dbs